src/components/ErrorComponent.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { translate } from '../plugins/i18n'
+
+const reload = () => {
+  window.location.reload()
+}
+
+const Oops = ({ t, errorType }) => (
```

Comment:
Not a big fan of the naming here.
Also,  `ErrorComponent` seems redundant as we already are into the `components` directory.